packages/template-retail-react-app/app/components/header/index.jsx

```diff
@@ -318,6 +339,9 @@ const Header = ({
                         {...styles.icons}
                         onClick={onMyCartClick}
                     />
+                    <HideOnDesktop display={{base: 'contents', lg: 'none'}}>
```

Comment:
🤔 what does `display: contents` do?

It actually got replaced by another more-specific css selector. So `display: none` is what actually got used. 

![Arc 2024-08-09 at 10 24 02](https://github.com/user-attachments/assets/2106c41a-2fd7-4856-ad43-c17ed44bd0fd)

Reply:
That's the expected behavior on the desktop screen. Check the styles used on mobile, and you'll see `display: contents` on mobile.